Crashlytics/Shared/FIRCLSConstants.m

```diff
@@ -13,13 +13,14 @@
 // limitations under the License.
 
 #import "Crashlytics/Shared/FIRCLSConstants.h"
+#import <FirebaseCore/FIRVersion.h>
 
 #define STR_HELPER(x) #x
```

Comment:
Delete these two lines

Reply:
I think these are still needed, but deleted the ones in FIRCLSDefines.h